README.md

```diff
@@ -11,6 +11,88 @@ more cloud vendors:
 
 - Amazon Web Services (AWS): go to the [aws](aws) folder for more details.
 
+## Prerequisites
+
+1. Terraform installed
+2. AWS CLI set up with appropriate permissions
```

Comment:
Can we double check this (tomorrow) is needed? AWS CLI is not the only way to configure the credentials, here are some other ways, where `aws` CLI is not explicitly mentioned.

https://registry.terraform.io/providers/hashicorp/aws/latest/docs#environment-variables